Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/CodeUtils.kt

```diff
@@ -44,6 +44,20 @@ fun String.asSimpleName(): String = if (this in kotlinKeywords || this.contains(
     this
 }
 
+/**
+Yet another mangler, particularly to avoid secondary clash, e.g. when a property in prototype (interface)
```

Comment:
Are secondary clashes still relevant here?

Reply:
Yes. We try to avoid another clash by applying characters which are not typical for manually written code - that simple. I let this comment to make clear the difference in rationale with `String.mangled`in `ObjCMethod.getKotlinParameterNames`